app/controllers/spree/credit_cards_controller.rb

```diff
@@ -27,6 +27,7 @@ def update
       authorize! :update, @credit_card
 
       if @credit_card.update(credit_card_params)
+        remove_shop_authorizations if credit_card_params["is_default"]
```

Comment:
This new behavior should be covered somewhere in the spec for this controller.

Reply:
👍 32cb63c448dcdeffe246ff0881b79cbd39417d75

Reply:
Great :+1: